snuba/subscriptions/data.py

```diff
@@ -383,6 +383,7 @@ def build_request(
                 metrics.increment("snql.subscription.delegate.error")
             logger.warning(
                 f"failed snql subscription: {e}",
+                exc_info=e,
```

Comment:
As I mentioned yesterday on Nikhar's PR, did you test that the sentry integration works properly when passing the exception to `exc_info` instead of just setting it to True ? I don't think we have any case in the code base.

Reply:
I tested it, it works fine.